vizro-core/tests/unit/vizro/models/_components/test_table.py

```diff
@@ -146,62 +146,77 @@ def test_pre_build_actions_underlying_table_id(self, dash_data_table_with_id, fi
 
 class TestBuildTable:
     def test_table_build_mandatory_only(self, standard_dash_table, gapminder):
-        table = vm.Table(id="text_table", figure=standard_dash_table)
+        table = vm.Table(figure=standard_dash_table)
```

Comment:
Remember we don't need to use the `id` everywhere. The `keys_to_strip` argument is exactly for this case. Here we want to use mandatory arguments, so no need to specify id. 

There is a separate test for testing the ID in this file as well.